ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -55,6 +55,11 @@ tests:
   openshift_installer:
     cluster_profile: gcp
     upgrade: true
+- as: e2e-sdn-step-registry
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
```

Comment:
@pecameron you can't copy what the non-step-registry jobs do (eg 'commands' and 'openshift_installer'). See the e2e-ovn-step-registry job that Jacob added right below yours for how step registry jobs work.

Reply:
fixed